### PR TITLE
Document approval process in `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by g1zzm0**
Currently, the process for approving pull requests has not been
documented. Though the process is not completely determined, and is 
still being discussed, we note down in this patch the bits that are
currently in place.

review-requested: @mristin

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,3 +49,22 @@ To validate the example data against the schemas, call:
...
+| Topic | Path | Reviewer |
+| ------------- | ------------- | ------------- |
+| JSON | [schemas/json/](schemas/json/) | @Manu3756 @torben.deppe |
+| RDF | [schemas/rdf/](schemas/rdf/) | @sebbader @changqin26 |
+| XML | [schemas/xml/](schemas/xml/) | @Manu3756 @JoergWende |
+| XMI | [schemas/xmi/](schemas/xmi/) | @BirgitBoss |
+| aas-specs-repo | [aas-specs/](https://github.com/admin-shell-io/aas-specs) | @BirgitBoss @mristin |
+
+
+Major changes must first be reviewed and approved by the joint working group of the [Platform Industrie 4.0](http://www.plattform-i40.de). and [IDTA](https://industrialdigitaltwin.org/).
```
Comment (mristin) [inline CONTRIBUTING.md:69]:
```suggestion
Major changes must first be reviewed and approved by the joint working group of the [Platform Industrie 4.0](http://www.plattform-i40.de) and [IDTA](https://industrialdigitaltwin.org/).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -49,3 +49,22 @@ To validate the example data against the schemas, call:
...
+Afte the approval the pull request can be merged in to the repository.
```
Comment (mristin) [inline CONTRIBUTING.md:70]:
```suggestion
Afte the approval the pull request is merged into the repository by the responsible reviewer.
```

**Review by mristin — APPROVED** [review]
LGTM, please see the minor suggestions. Please let me know when it is final so that I can merge it in.

File: CONTRIBUTING.md (hunk 2 above)
Comment (g1zzm0) [reply CONTRIBUTING.md:70]: Maybe reviewer can't push into main?

**Comment by mristin** [comment]
> Maybe reviewer can't push into main?

Correct. Maybe write that an editor needs to be finally pinged and asked to
do the final merge into master?

**Comment by g1zzm0** [comment]
> Maybe reviewer can't push into main?
> Correct. Maybe write that an editor needs to be finally pinged and asked to do the final merge into master?

Is there a good way to link the GitHub users?
The usage of @[username] dos not link the GitHub accounts.

**Comment by mristin** [comment]
> Is there a good way to link the GitHub users?

You mean in the markdown documents? Maybe linking to their personal pages
such as https://github.com/mristin?

**Comment by mristin** [comment]
LGTM. Merge?

**Comment by g1zzm0** [comment]
Okay.

**Review by mristin — COMMENTED** [review]
LGTM

**Pull request merged**